src/components/Contacts/ContactDetails/ContactDonationsTab/PartnershipInfo/EditPartnershipInfoModal/EditPartnershipInfoModal.test.tsx

```diff
@@ -19,7 +19,30 @@ import {
 } from '../../ContactDonationsTab.generated';
 import { EditPartnershipInfoModal } from './EditPartnershipInfoModal';
 
+jest.mock('next/router', () => ({
```

Comment:
Personally, I think it's better to use `<TestRouter>` instead of mocking `useRouter` so that our test code is closer to our application code.

Reply:
I'll use it